libs/legi-shared/autocomplete/autocomplete-multi/autocomplete-multi.component.ts

```diff
@@ -88,6 +88,10 @@ export class AutocompleteMultiComponent<T> extends ControlComponent<T[]> impleme
   hint?: string;
   @Input()
   removeDisabledOptions = true;
+  @Input()
+  allowDropdownTrigger: boolean = false;
+  @Input()
+  optionType: "lookup" | 'raw' = "lookup";
```

Comment:
issue being, when using a autocomplete multi, it was adding "create" to the mat-chip